src/components/Course/index.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { ImPriceTags } from "react-icons/im";
+import { registerCourse } from "../../apis";
+import { toast } from "react-toastify";
+import { useNavigate } from "react-router-dom";
+import styles from "./style.module.css";
+
+function Course({ course, i }) {
```

Comment:
What is "i"?
Need to name it clearly.

Reply:
Maybe it's the stat, I just pressed Gracie. kaka